java/sidecar/src/main/java/com/instaclustr/cassandra/sidecar/cassandra/CassandraModule.java

```diff
@@ -17,7 +23,25 @@ public CassandraModule(final MBeanServerConnection mBeanServerConnection) {
 
     @Provides
     @Singleton
-    StorageServiceMBean storageServiceMBeanProvider() {
+    public StorageServiceMBean storageServiceMBeanProvider() {
         return JMX.newMBeanProxy(mBeanServerConnection, CassandraObjectNames.STORAGE_SERVICE_MBEAN_NAME, StorageServiceMBean.class);
     }
+
+    @Provides
```

Comment:
These other providers should be moved to a separate module (such as `OperationsModule`)